app-modules/form/src/Filament/Resources/FormResource/Pages/ManageFormSubmissions.php

```diff
@@ -153,7 +153,7 @@ public static function getNavigationItems(array $urlParameters = []): array
         $ownerRecord = $urlParameters['record'];
 
         /** @var Form $ownerRecord */
-        $formSubmissionsCount = Cache::tags('form-submission-count')
+        $formSubmissionsCount = Cache::tags('{{form-submission-count}}')
```

Comment:
This should only have one set of `{}` around this.

In the example you are copying from, it only has the double `{}` because it is within a string with a variable in it and String interpolation.